autoextract_spiders/settings.py

```diff
@@ -42,10 +42,12 @@
 # Enable or disable downloader middlewares
 # See http://scrapy.readthedocs.org/en/latest/topics/downloader-middleware.html
 DOWNLOADER_MIDDLEWARES = {
+    'scrapy.downloadermiddlewares.useragent.UserAgentMiddleware': None,
     'scrapy_crawlera.CrawleraMiddleware': 300,
     'scrapy_count_filter.middleware.GlobalCountFilterMiddleware': 541,
     'scrapy_count_filter.middleware.HostsCountFilterMiddleware': 542,
     'scrapy_autoextract.middlewares.AutoExtractMiddleware': 543,
+    'autoextract_spiders.middlewares.RandomUserAgentMiddleware': 998,
```

Comment:
I'm not sure what it affects, but a priority for `scrapy.downloadermiddlewares.useragent.UserAgentMiddleware` is 500 by default. So when FAKEUSERAGENT_ENABLED is False, RandomUserAgentMiddleware (which essentially becomes UserAgentMiddleware) uses a different priority. 

I don't see any immediate problems with it, but could you please check that it doesn't affect things? Alternatively, does it work with the same priority as built-in middleware, to be on a safe side?

Reply:
You're right, I missed it :100: It doesn't look like a big deal, but yeah, let's use the default priority to avoid unwanted side-effects.